crates/api_models/src/payouts.rs

```diff
@@ -537,6 +557,36 @@ pub struct PayoutCreateResponse {
     pub phone_country_code: Option<String>,
 }
 
+/// The payout method information for response
+#[derive(Debug, Clone, Deserialize, Serialize, ToSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum PayoutMethodDataResponse {
+    Card(Box<CardPayoutResponse>),
+    Bank(Box<BankPayoutResponse>),
+    Wallet(Box<WalletPayoutResponse>),
```

Comment:
These can directly be CardAdditionalData, BankAdditionalData and WalletAdditionalData

We had added `details` and we were flattening it for payments for backwards compatibility only